app/templates/test.twig

```diff
@@ -0,0 +1,15 @@
+{% extends "layouts/basic.layout.twig" %}
```

Comment:
Load the `twig.php` URL to see how this fits everything together- `extends` the basic layout template, but inserts things into blocks (below)